### PR TITLE
Adds the Emergency Tether to the Engineering Modsuit

### PR DESCRIPTION
**Pull request by TheSS13Melon**
## About The Pull Request

Adds Emergency Tethers to the engineering modsuit.

## How This Contributes To The Skyrat Roleplay Experience

Tethers are pretty much unused because they're quickly phased out by the jetpack. This adds them to engineering suits as engineering uses their modsuits in EVA more often than other groups (notably for solars at the start) so they have at least a little bit of use before they're replaced by jetpacks.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image

</details>

## Changelog

:cl:
qol: Adds the Emergency Tether to Engineering Modsuits
/:cl:

### CODE REVIEW

**Comment by Plum-but-gayer** [comment]
As a longtime engineer player/modsuit enjoyer 100% agree with this, tethers never have a use case outside of a very small 20 minute window and the fuss to install them is rarely worth it. Engineers having some extra EVA mobility roundstart (as they are the only people who really enter EVA roundstart) makes sense and is good. +1

**Comment by ghost** [comment]
Proof of Testing image is broken: make sure you put Whitespace between it and any surrounding text

**Review by lessthnthree — APPROVED** [review]

File: code/modules/mod/mod_types.dm [hunk 1]
```diff
@@ -52,10 +52,12 @@
 		/obj/item/mod/module/welding,
 		/obj/item/mod/module/rad_protection,
 		/obj/item/mod/module/flashlight,
+		/obj/item/mod/module/tether,
 		/obj/item/mod/module/magboot,
 	)
 	default_pins = list(
 		/obj/item/mod/module/magboot,
+		/obj/item/mod/module/tether,
```
Comment (lessthnthree) [inline code/modules/mod/mod_types.dm]: Need to mark your Skyrat edits on non-modular PRs.

**Review by lessthnthree — CHANGES_REQUESTED** [review]

File: code/modules/mod/mod_types.dm (hunk 1 above)
Comment (GoldenAlpharex) [inline code/modules/mod/mod_types.dm]:
```suggestion
		/obj/item/mod/module/tether, // SKYRAT EDIT ADDITION - Adds the Emergency Tether to the roundstart engineering MODsuits
		/obj/item/mod/module/magboot,
	)
	default_pins = list(
		/obj/item/mod/module/magboot,
		/obj/item/mod/module/tether, // SKYRAT EDIT ADDITION - Adds the Emergency Tether to the roundstart engineering MODsuits
```

**Review by GoldenAlpharex — APPROVED** [review]
I approve of this change.

**Comment by ghost** [comment]
Would TG accept this? It's a pretty simple QoL change

**Review by lessthnthree — APPROVED** [review]

**Comment by Paxilmaniac** [comment]
> Would TG accept this? It's a pretty simple QoL change

This certainly seems like something that should have been tried upstream first.

**Pull request merged**